### PR TITLE
20241026 wasm exorcism

### PR DESCRIPTION
**Pull request by Firstyear**
# Change summary

- Liberal party got in, cut everything.

Checklist

- [x] This PR contains no AI generated code
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

**Review by yaleman — DISMISSED** [review]
Awwww yisss.

**Review by yaleman — APPROVED** [review]

File: server/lib/benches/image_benches.rs [hunk 1]
```diff
@@ -1,134 +1,135 @@
-/// This file contains benchmarks for the image module so we can work out the best order to run things in
-use std::time::Duration;
+// /// This file contains benchmarks for the image module so we can work out the best order to run things in
 
-use criterion::{black_box, criterion_group, criterion_main, Criterion};
-use kanidmd_lib::valueset::image::jpg;
-use kanidmd_lib::valueset::image::png;
+// use std::time::Duration;
```
Comment (CEbbinghaus) [inline server/lib/benches/image_benches.rs:3]: Did we want to delete instead of commenting out?

**Review by CEbbinghaus — APPROVED** [review]
Good stuff. Finally, some good commits

File: server/lib/benches/image_benches.rs (hunk 1 above)
Comment (Firstyear) [reply server/lib/benches/image_benches.rs:3]: Not even a 1% gain in lines reduced :P

**Comment by Firstyear** [comment]
@CEbbinghaus They're all good commits.

**Comment by CEbbinghaus** [comment]
Not saying they aren't. Just quoting Gordon ramsey
image
:P

**Comment by Firstyear** [comment]
https://knowyourmeme.com/memes/theyre-good-dogs-brent

**Pull request merged**